lib/src/wallet/keypair.dart

```diff
@@ -18,7 +19,7 @@ class KeyPair {
     return privateKey;
   }
 
-  Future<List<int>> getPublicKey() async {
+  Future<List<int>> getPublicKey([bool verify = false]) async {
```

Comment:
What's the purpose of this positional parameter?

Reply:
I added this for the ledger impl. In case the retrival of the public key needs to be verified by the user. It's specific to the ledger impl. Maybe it is better to move it out of the interface, into the specific ledger class impl.

Reply:
I think it would make sense to move it away from this interface since it's more of a UI level feature requiring input from the user.